tests/pl/homeassistant_HassTurnOn.yaml

```diff
@@ -10,6 +10,7 @@ tests:
   - sentences:
       - "Zapal lampka nocna"
       - "Włącz lampka nocna"
+      - "Zaświeć lampka nocna"
```

Comment:
Correct forms are missing "Włącz lampkę nocną" and others.

Reply:
Will improve, thank you 😃